helm/alfresco-content-services/Chart.lock

```diff
@@ -5,6 +5,9 @@ dependencies:
 - name: postgresql
   repository: oci://registry-1.docker.io/bitnamicharts
   version: 12.8.5
+- name: postgresql
+  repository: oci://registry-1.docker.io/bitnamicharts
+  version: 12.8.5
```

Comment:
I wonder how helm could work with that

Reply:
From what I see it make Helm download the same chart twice which is indeed sub-optimal but this file is autogenerated so...